README.md

```diff
@@ -288,6 +273,19 @@ sudo apt install apache2-utils
 
 ## Server administration
 
+### Deploying a new version
```

Comment:
This is already on the main branch now, but at the very end. I guess it would be good to improve that version.

Reply:
Looks like I accidentally duplicated it during a rebase. I'll merge the sections into one.